examples/Text Editor v1.0/TextEditor_ReadMe.txt

```diff
@@ -0,0 +1,12 @@
+My First Project in PyQt5. 
+A simple Text Editor which have these functionalities:
+
+1. Making a new Txt File
+2. Openig a existing Txt File
+3. Saving Txt File
+4. Basic operations: Undo/Redo and Cut/Copy/Paste - added by coding, instead of using predefined functions
+5. Exit function. 
+6. Save on Exit
+7. Help Menu 
+
+Modifications will be done in this Text Editor to add more functioanlities like Text Formatting, Find and Replae. etc..
```

Comment:
sloppy! functioanlities? come one now, it's *functionalities*. At the end, it's
[comma] etc[dot] --> ..., Find and Replace, etc.

Reply:
Yes, these errors are because of the hurry! Wierd spelling mistake. It is a typing error. Thanks !